### PR TITLE
[Bug] Timezone awareness for session start time checks

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
## Motivation

Discovered by @garrettmflynn 

Drafting one possible solution

### CODE REVIEW

File: src/nwbinspector/checks/nwbfile_metadata.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def check_session_start_time_old_date(nwbfile: NWBFile):
     session_start_time = nwbfile.session_start_time
     dummy_time = datetime(1980, 1, 1)
     if session_start_time.tzinfo is not None:
-        dummy_time = dummy_time.astimezone()
+        dummy_time.replace(tzinfo=session_start_time.tzinfo)
```
Comment (CodyCBakerPhD) [inline src/nwbinspector/checks/nwbfile_metadata.py]:
```suggestion
        dummy_time = dummy_time.replace(tzinfo=session_start_time.tzinfo)
```

File: src/nwbinspector/checks/nwbfile_metadata.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def check_session_start_time_future_date(nwbfile: NWBFile):
     session_start_time = nwbfile.session_start_time
     current_time = datetime.now()
     if session_start_time.tzinfo is not None:
-        current_time = current_time.astimezone()
+        current_time.replace(tzinfo=session_start_time.tzinfo)
```
Comment (CodyCBakerPhD) [inline src/nwbinspector/checks/nwbfile_metadata.py]:
```suggestion
        current_time = current_time.replace(tzinfo=session_start_time.tzinfo)
```

**Comment by garrettmflynn** [comment]
Updated so that the timezone information is taking effect properly in the Guided Inspect page.

**Review by garrettmflynn — APPROVED** [review]
Looks good to me!

**Pull request merged**